bin/ci-reindex

```diff
@@ -2,21 +2,19 @@
 set -e
 . bin/ci-process-bootstrap
 
-./bin/console queue:clean
-# TODO: poll for queue:count == 0, with timeout
-sleep 5
+# Clean the queues
+./bin/ci-empty-queue-and-wait
 
-index_name=elife_search_$(date +%Y%m%d%H%M%S)
-./bin/console search:setup --index="$index_name" --delete
+./bin/console search:setup --index="ci-reindex" --delete
```

Comment:
I've no objection to removing the date and time from the index name. This could be useful if rerunning locally and debugging. Not necessary though.

Reply:
These changes should stop indexes accumulating as the scripts would never delete them. Insures that the tests start from a more consistent baseline.

Reply:
*ensures

There is a ./bin/console command called `index:delete:unused` that will "Delete all unused indexes". I tried running it recently and didn't see any change.

Handling accumulating indices is easy but debugging is typically hard. See how you go @nlisgo and we can always add a datestamp back.